src/OpenTelemetry.Instrumentation.AspNetCore/.publicApi/net7.0/PublicAPI.Unshipped.txt

```diff
@@ -12,9 +12,12 @@ OpenTelemetry.Instrumentation.AspNetCore.AspNetCoreInstrumentationOptions.Filter
 OpenTelemetry.Instrumentation.AspNetCore.AspNetCoreInstrumentationOptions.Filter.set -> void
 OpenTelemetry.Instrumentation.AspNetCore.AspNetCoreInstrumentationOptions.RecordException.get -> bool
 OpenTelemetry.Instrumentation.AspNetCore.AspNetCoreInstrumentationOptions.RecordException.set -> void
+OpenTelemetry.Instrumentation.AspNetCore.AspNetCoreMetricsInstrumentationOptions
```

Comment:
FYI in order to get the CI passing you'll need to update this file for the other targets as well (net6.0, netcoreapp3.1, netstandard2.0, & netstandard2.1).

Reply:
Updated to all targets